src/api/firebase.js

```diff
@@ -55,7 +55,7 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listId);
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
-	return console.log(listCollectionRef, {
+	return addDoc(listCollectionRef, {
```

Comment:
shouldn't make a difference, but I would wonder if you're supposed to `return await addDoc(...)` here